Storage Provider Incubation Center/README.md

```diff
@@ -154,7 +148,7 @@
 
```

Comment:
Candidates interested in the Venus Master Program need to show proof of contribution to the Filecoin network by participating as a storage provider themselves and have coding experience in the Filecoin network stack. The selected Master will provide consulting services and technical support to the participants of the incubation center program.  The Venus Master eventually will be able to lead his own cohorts.